backends/rapidpro/channel_log.go

```diff
@@ -120,28 +119,26 @@ func NewDBLogWriter(db *sqlx.DB, wg *sync.WaitGroup) *DBLogWriter {
 			defer cancel()
 
 			writeDBChannelLogs(ctx, db, batch)
-		}, time.Millisecond*500, 1000, wg),
+		}, 1000, time.Millisecond*500, 1000, wg),
 	}
 }
 
-func writeDBChannelLogs(ctx context.Context, db *sqlx.DB, logs []*dbChannelLog) {
-	for _, batch := range utils.ChunkSlice(logs, 1000) {
```

Comment:
no longer need to batch here since the batcher will only ever return batches <= 1000